### PR TITLE
METRON-2021 add screen and configuration to image

### PR DESCRIPTION
**Pull request by ottobackwards**
## testing

`./run_end_to_end.sh && scripts/docker_execute_shell.sh`
` screen`
`ctrl-a ctrl-d`
`exit`
`exit`
`scripts/docker_execute_shell.sh`
`screen -r`

You can also test just the screenrc without docker by going to the directory
and running `screen -c .screenrc

## Pull Request Checklist

Thank you for submitting a contribution to Apache Metron's Bro kafka writer plugin.

In order to streamline the review of the contribution we ask you follow these guidelines and ask you to double check the following:

### For all changes:
- [x] Is there a JIRA ticket associated with this PR? If not one needs to be created at [Metron Jira](https://issues.apache.org/jira/browse/METRON/?selectedTab=com.atlassian.jira.jira-projects-plugin:summary-panel).
- [x] Does your PR title start with METRON-XXXX where XXXX is the JIRA number you are trying to resolve? Pay particular attention to the hyphen "-" character.
- [x] Has your PR been rebased against the latest commit within the target branch (typically master)?

### For code changes:
- [-] Have you included steps to reproduce the behavior or problem that is being changed or addressed?
- [x] Have you included steps or a guide to how the change may be verified and tested manually?
- [x] Have you ensured that the full suite of tests and checks have been executed via:
  ```
  bro-pkg test $GITHUB_USERNAME/metron-bro-plugin-kafka --version $BRANCH
  ```
- [-] Have you written or updated unit tests and or integration tests to verify your changes?
- [-] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [-] Have you verified the basic functionality of the build by building and running locally with Apache Metron's [Vagrant full-dev environment](https://github.com/apache/metron/tree/master/metron-deployment/development/centos6) or the equivalent?

### CODE REVIEW

File: docker/containers/bro-localbuild-container/.screenrc [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# terminfo and termcap for nice 256 color terminal
+# allow bold colors - necessary for some reason
+attrcolor b ".I"
+# tell screen how to set colors. AB = background, AF=foreground
```
Comment (JonZeolla) [inline docker/containers/bro-localbuild-container/.screenrc]: Can we add whitespace between the config lines and the blocks of comments, and clean up the extraneous whitespace at the end?

Also, we should increase `defscrollback`, I usually do 20000.

**Comment by JonZeolla** [comment]
Also, what about something like:
```# Caption the screen sessions
caption always "%{= Wk}%-w%{= Bw}%n %t%{-}%+w %-="```

File: docker/containers/bro-localbuild-container/.screenrc (hunk 1 above)
Comment (JonZeolla) [reply docker/containers/bro-localbuild-container/.screenrc]: Also, what about something like:
```
# Caption the screen sessions
caption always "%{= Wk}%-w%{= Bw}%n %t%{-}%+w %-="
```

Comment (ottobackwards) [reply docker/containers/bro-localbuild-container/.screenrc]: My status line includes captioning of windows, this statement conflicts.

**Comment by JonZeolla** [comment]
+1 by inspection

**Pull request closed without merging**